cli/batch/cli.py

```diff
@@ -0,0 +1,76 @@
+"""
+Provide implementation of the command line interface's batch commands.
+"""
+import sys
+import re
+
+import asyncio
+import click
+from remme import Remme
+
+from cli.batch.help import (
+    GET_BATCH_ID_ARGUMENT_HELP_MESSAGE,
+    GET_BATCH_START_ARGUMENT_HELP_MESSAGE,
+    GET_BATCH_LIMIT_ARGUMENT_HELP_MESSAGE,
+    GET_BATCH_HEAD_ARGUMENT_HELP_MESSAGE,
+    GET_BATCH_REVERSE_ARGUMENT_HELP_MESSAGE,
+)
+from cli.batch.service import Batch
+from cli.constants import (
+    BATCH_ID_REGEXP,
+    FAILED_EXIT_FROM_COMMAND,
+    NODE_URL_ARGUMENT_HELP_MESSAGE,
+)
+
+loop = asyncio.get_event_loop()
+
+
+@click.group('batch', chain=True)
+def batch_commands():
+    """
+    Provide commands for working with batches.
+    """
+    pass
+
+
+@click.option('--id', type=str, required=False, help=GET_BATCH_ID_ARGUMENT_HELP_MESSAGE)
+@click.option('--start', type=str, required=False, help=GET_BATCH_START_ARGUMENT_HELP_MESSAGE)
+@click.option('--limit', type=int, required=False, help=GET_BATCH_LIMIT_ARGUMENT_HELP_MESSAGE)
+@click.option('--head', type=str, required=False, help=GET_BATCH_HEAD_ARGUMENT_HELP_MESSAGE)
+@click.option('--reverse', type=str, required=False, help=GET_BATCH_REVERSE_ARGUMENT_HELP_MESSAGE)
+@click.option('--node-url', type=str, help=NODE_URL_ARGUMENT_HELP_MESSAGE)
+@batch_commands.command('get-list')
+def get_list(id, start, limit, head, reverse, node_url):
+    """
+    List batches.
+    """
+    for batch_id in (id, start, head):
+        if batch_id is not None and re.match(pattern=BATCH_ID_REGEXP, string=batch_id) is None:
+            click.echo('The following batch id `{batch_id}` is not valid.'.format(batch_id=id))
+            sys.exit(FAILED_EXIT_FROM_COMMAND)
+    if limit is not None and limit < 0:
```

Comment:
Blank line between 50 and 51 code lines.

Reply:
https://github.com/dmytrostriletskyi/nimble-python-code-style-guide#indentions

Reply:
Done.